core/state_processor_test.go

```diff
@@ -383,15 +383,23 @@ func TestProcessStateless(t *testing.T) {
 	defer blockchain.Stop()
 	txCost1 := params.WitnessBranchWriteCost*2 + params.WitnessBranchReadCost*2 + params.WitnessChunkWriteCost*3 + params.WitnessChunkReadCost*10 + params.TxGas
 	txCost2 := params.WitnessBranchWriteCost + params.WitnessBranchReadCost*2 + params.WitnessChunkWriteCost*2 + params.WitnessChunkReadCost*10 + params.TxGas
-	blockGasUsedExpected := txCost1*2 + txCost2
+	contractCreationCost := uint64(70919)
```

Comment:
hardcoded value, break it into its components

Reply:
this is the last thing to be figured out before the PR can be merged, pending that the tests pass